pagopa/io-app
# chore(IT Wallet): [SIW-1550] Add CTA to view/download PDF in the European Health Insurance card's details screen

### ⚠️ Depends on https://github.com/pagopa/io-app/pull/6154

## Short description
This PR adds the credential's attachments screen, reachable with a CTA from the credential's detail screen.  From the attachments screen the user can download or share the document.

## List of changes proposed in this pull request
- Added `ItwPresentationCredentialAttachmentScreen`
- Added the ability to set a footer CTA in `ItwPresentationDetailsScreenBase` component
- Added `getCtaProps` function which return the CTA to display based on the credential's claims.
- Removed `Readex` font in credential screen header 

## How to test
Navigate to the European Health Insurance card's details screen, tap on the **View PDF document** CTA. Check that the document is correctly visible.
Tap on **Download or share** CTA, check that the document can be saved to the device or shared via other apps.

## Preview

https://github.com/user-attachments/assets/faeab357-decf-4db6-9a9d-54024a1931a1
